component/libfuse/libfuse_handler_test.go

```diff
@@ -60,7 +60,7 @@ func (suite *libfuseTestSuite) TestDefault() {
 func (suite *libfuseTestSuite) TestConfig() {
 	defer suite.cleanupTest()
 	suite.cleanupTest() // clean up the default libfuse generated
-	config := "allow-other: true\nread-only: true\nlibfuse:\n  default-permission: 0777\n  attribute-expiration-sec: 60\n  entry-expiration-sec: 60\n  negative-entry-expiration-sec: 60\n  fuse-trace: true\n"
```

Comment:
any reason to remove this default-permission thing ?

Reply:
I added a test below for default permission  TestConfigDefaultPermission.

I was trying to hit the different permissions cases with these 3 tests., setting default permissions with allow other is actually not useful based on our logic https://github.com/Azure/azure-storage-fuse/blob/main/component/libfuse/libfuse.go#L161